### PR TITLE
Refactor lucid vision driver

### PR DESCRIPTION
**Pull request by Zeysthingz**
Changed camera topic to clarify which camera is which one and deleted comressed and rectified image publishers because we use image proc currently in container and it provides this topics

### CODE REVIEW

File: src/arena_camera_node.cpp [hunk 1]
```diff
@@ -24,12 +24,6 @@ ArenaCameraNode::ArenaCameraNode(rclcpp::NodeOptions node_options)
   m_publisher = this->create_publisher<sensor_msgs::msg::Image>(
     create_camera_topic_name(camera_settings.get_camera_name()) + "/image",
     rclcpp::QoS{1}.reliable());
-  m_rect_publisher = this->create_publisher<sensor_msgs::msg::Image>(
```
Comment (kaancolak) [inline src/arena_camera_node.cpp:27]: Rectified ve compressed image'i enable disable edebiliyor olmamiz lazim parametre dosyasindan, publisher yine de olusuyor topic ismi gorunuyorsa publish etmeden once yaptigimiz kontrolu, publisher olusturulmadan da once yapabiliriz.

File: param/back_left_camera.param.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /**:
   ros__parameters:
-    camera_name: "camera_5"
+    camera_name: "back_left_camera"
```
Comment (kaancolak) [inline param/back_left_camera.param.yaml:3]: Back yerine rear kullanabilir miyiz? Lidar topiclerini de bu sekilde olusturduk

**Review by kaancolak — COMMENTED** [review]
I left some comments

File: src/arena_camera_node.cpp (hunk 1 above)
Comment (Zeysthingz) [reply src/arena_camera_node.cpp:27]: Done

**Pull request closed without merging**